src/utils/allocator_default.h

```diff
@@ -1,10 +1,14 @@
 #ifndef ALLOCATOR_DEFAULT_H
 #define ALLOCATOR_DEFAULT_H
 
```

Comment:
Should you not define allocator_n64 instead?

Reply:
Not sure, I was suggesting a platform.c but that was shutdown.

Reply:
Keeping the implementation in a header file, doesn't allow me to override this for just the N64.

Reply:
1. Copy allocator_default.h to allocator_n64.h
2. Make allocator_n64.h handle allocations in n64 style
3. Add ifdef to allocator.h to use allocator_n64.h instead (where there is comment "Add ifdefs here to include platform-specific allocators"). 
4. Import allocator.h

This should work fine ?